package.json

```diff
@@ -78,13 +78,13 @@
     "uuid": "^3.0.1"
   },
   "devDependencies": {
-    "@ciscospark/eslint-config": "0.7.87",
+    "@ciscospark/eslint-config": "^0.7.87",
```

Comment:
Just for the eslint-config since it's a piece of tooling and does not disrupt feature changes.